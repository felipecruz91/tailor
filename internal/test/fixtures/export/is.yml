apiVersion: v1
kind: List
items:
- apiVersion: image.openshift.io/v1
  kind: ImageStream
  metadata:
    annotations:
      description: Keeps track of changes in the application image
      openshift.io/image.dockerRepositoryCheck: 2018-08-07T12:32:24Z
      openshift.io/generated-by: OpenShiftNewApp
      kubectl.kubernetes.io/last-applied-configuration: '{"apiVersion":"v1","kind":"ImageStream","metadata":{"annotations":{"foo":"bar"}}}'
    creationTimestamp: null
    generation: 560
    labels:
      app: foo-bar
    name: bar
  spec:
    dockerImageRepository: bar
    lookupPolicy:
      local: false
    tags:
    - annotations: null
      from:
        kind: ImageStreamImage
        name: bar@sha256:4e418dd975063c99f52d0d17076b54e38186a90deb34cd5c502ed045e9c385da
      generation: 560
      importPolicy: {}
      name: latest
      referencePolicy:
        type: Source
  status:
    dockerImageRepository: ""
