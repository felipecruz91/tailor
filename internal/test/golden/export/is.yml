apiVersion: template.openshift.io/v1
kind: Template
objects:
- apiVersion: image.openshift.io/v1
  kind: ImageStream
  metadata:
    annotations:
      description: Keeps track of changes in the application image
      openshift.io/generated-by: OpenShiftNewApp
    labels:
      app: foo-bar
    name: bar
  spec:
    dockerImageRepository: bar
    lookupPolicy:
      local: false
